docs/azure/services/Microsoft_Purview_Information_Protection_Overview.md

```diff
@@ -0,0 +1,96 @@
+
+Enterprises dealing with sensitive information such as financial data, healthcare-related and other personal records are required to adequately record and govern how this data is handled. As part of this process, the system should classify, label and control access to data in accordance with regulatory requirements. 
+
+Due to the nature of large organisations, various bits of sensitive data can end up scattered across on-premise servers, employee devices, cloud-native applications, third-party applications and many other resources. An illustrative diagram from Microsoft's [documentation](https://learn.microsoft.com/en-us/microsoft-365/compliance/information-protection?view=o365-worldwide) shows the high-level capabilities offered by Microsoft Purview Information Protection.
+
+![](../images/PurviewOverview.png)
+
+## Data Discovery
+
+There are three main ways data can be discovered using Purview:
+- Manually - done by users
+- Automated pattern recognition - Purview has a concept of Sensitive information types (SITs) which is used to automatically discover sensitive data based on built-in or custom declared patterns. Microsoft has created approximately 200 built-in patterns to recognise different types of card, healthcare, identity information used across different regions. A list of these patterns and their definitions can be found [here](https://learn.microsoft.com/en-gb/microsoft-365/compliance/sensitive-information-type-entity-definitions?view=o365-worldwide).
+- Machine learning - Pre-trained and custom trainable classifiers are a Purview feature that is used to discover sensitive data based on its content by modelling it based on a sample set of true positive matches to sensitive data to use as training. Microsoft has several pre-trained classifiers and their definitions can be seen [here](https://learn.microsoft.com/en-us/microsoft-365/compliance/classifier-tc-definitions?view=o365-worldwide#trainable-classifiers-definitions). Custom classifiers requires an organisation to prepare a dataset of true positives for a given type of sensitive data and would need some level of training to increase its accuracy before it can be used and published. More information can be found [here](https://learn.microsoft.com/en-gb/microsoft-365/compliance/classifier-learn-about?view=o365-worldwide)
+
+## Data Protection
+
+Once sensitive data is discovered, it needs to be labelled with appropriate sensitivity and retention labels. These labels will configure the classification of that data and therefore prepare it to be protected and governed according to existing compliance and business needs. 
+
+### Sensitivity Labels
+
+New and existing sensitivity labels can be found under the "Information Protection" tab on the compliance portal.
+
+![](../images/InformationProtection.png)
+
+The following actions can be achieved by labelling the data:
+- Encrypt emails, meeting invites, and documents to prevent unauthorized people from accessing this data
```

Comment:
Do meeting invites also get marked by RMS? From what I'm aware they aren't and I can see a closed Issue discussing this also in 2021 but maybe it has changed https://github.com/MicrosoftDocs/Azure-RMSDocs/issues/1363